fe/fe-core/src/main/java/com/starrocks/sql/ast/UserIdentity.java

```diff
@@ -62,6 +62,8 @@ public class UserIdentity implements ParseNode, Writable, GsonPostProcessable {
     private String host;
     @SerializedName("isDomain")
     private boolean isDomain;
+    @SerializedName("eph")
+    private boolean ephemeral;
```

Comment:
Please add a comment explaining the use of "ephemeral"

Reply:
done